src/pymor/analyticalproblems/thermalblock.py

```diff
@@ -34,62 +33,48 @@ class ThermalBlockProblem(EllipticProblem):
            |        |        |        |
            ----------------------------
 
-    The Problem is implemented as an |EllipticProblem| with the
-    characteristic functions of the blocks as `diffusion_functions`.
-
     Parameters
     ----------
     num_blocks
         The tuple `(nx, ny)`
     parameter_range
         A tuple `(μ_min, μ_max)`. Each |Parameter| component μ_ij is allowed
         to lie in the interval [μ_min, μ_max].
-    rhs
-        The |Function| f(x, μ).
```

Comment:
Since one now can simply write `thermal_block_problem().with_(rhs=...)` I see no need to have this as an extra argument.